pglookout/pglookout.py

```diff
@@ -109,7 +105,9 @@ def load_config(self, _signal=None, _frame=None):
                 self.config = json.load(fp)
         except Exception as ex:  # pylint: disable=broad-except
             self.log.exception("Invalid JSON config, exiting")
-            self.stats.unexpected_exception(ex, where="load_config")
+            stats = getattr(self, "stats")
```

Comment:
`getattr` without a default value still fails with `AttributeError`, like `self.stats` would.
If the attribute is always there but can be `None`, then `if self.stats:` is enough.
If the attribute can be completely missing, then you need to add a default value to the `getattr` call.

I'd rather have a `self.stats` that can be `None` than an attribute that can be missing, to make "Find Usages" more reliable and not miss the `getattr` line.

Reply:
Oh well...

I think I would have liked to use use this pattern, but either pylint or pytype gave me the finger for it:

```
if hasattr(self, "stats") and self.stats:
     self.stats.unexpected_exception()
```

Will try again...

Reply:
Meh: 
```
************* Module pglookout.pglookout
pglookout/pglookout.py:109:19: E0203: Access to member 'stats' before its definition line 115 (access-member-before-definition)
pglookout/pglookout.py:110:20: E0203: Access to member 'stats' before its definition line 115 (access-member-before-definition)
```

Seems pylint is a bit stupid here... https://github.com/PyCQA/pylint/issues/2315#issuecomment-628331369 -> added the pylint ignores for now...

Reply:
Maybe I'm missing something here, why do you want to remove the `stats` attribute from __init__ ?